### DISCUSSION TITLE
swtiching from PEPS to GEODES ?

### DISCUSSION THREAD

**Discussion by opseasoi** [CS-SI/eodag#1184] [General]
Hi, 
Are you planning to integrate into EODAG the new CNES provider GEODES, replacing PEPS provider by the end of this month ?
https://geodes.cnes.fr/api/
Regards,
Pascal.

**Comment by sbrunato** [comment]
Hello @opseasoi, the switch will happen but I am not able to tell you when. Everything should work only with configuration, using existing plugins.
A Pull Request would be welcome if you need the support of this new provider soon

**Reply by sbrunato** [reply]
related to #1216

**Reply by opseasoi** [reply]
That will be great !
Thank you

**Comment by aaiche-meoss** [comment]
Hello @sbrunato , PEPS wouldn't be available from October.
Unless I'm mistaken, I see that you haven't yet updated the eodag tool. Could we know when it woud be updated? Thank you.

**Reply by sbrunato** [reply]
Hello @aaiche-meoss , `geodes` is now available on the `develop` branch, and will be included in the next release. See https://github.com/CS-SI/eodag/pull/1357

**Comment by opseasoi** [comment]
Good news, thanks !
Pascal.

**Comment by gg-bb** [comment]
Hello sbrunato,
I tried to use geodes provider with the latest version of eodag (eodag-3.0.1). I made an account on the geodes plateforme and generated an APIKEY (and created the global variable accordingly) following the tutorial you provided. However I have an error message that seems to be related  package version conflict; I tried multiple eodag uninstall and reinstall, updated the eadag.yml with geodes provider but I can't figure out the solution.

Any help would be higly appreciated, here the commands:
Best regards,
GB

```py
from eodag import EODataAccessGateway

dag=EODataAccessGateway()

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/-----/anaconda3/lib/python3.11/site-packages/eodag/api/core.py", line 163, in __init__
    self._plugins_manager = PluginManager(self.providers_config)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/-----/anaconda3/lib/python3.11/site-packages/eodag/plugins/manager.py", line 108, in __init__
    entry_point.load()
  File "/home/-----/anaconda3/lib/python3.11/site-packages/pkg_resources/__init__.py", line 2517, in load
    self.require(*args, **kwargs)
  File "/home/-----/anaconda3/lib/python3.11/site-packages/pkg_resources/__init__.py", line 2540, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/-----/anaconda3/lib/python3.11/site-packages/pkg_resources/__init__.py", line 829, in resolve
    dist = self._resolve_dist(
           ^^^^^^^^^^^^^^^^^^^
  File "/home/-----/anaconda3/lib/python3.11/site-packages/pkg_resources/__init__.py", line 875, in _resolve_dist
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (shapely 2.0.5 (/home/-----/anaconda3/lib/python3.11/site-packages), Requirement.parse('shapely>=2.0.6'))
```

**Reply by sbrunato** [reply]
Hello @gg-bb ,
> pkg_resources.VersionConflict: (shapely 2.0.5 (/home/-----/anaconda3/lib/python3.11/site-packages), Requirement.parse('shapely>=2.0.6'))

`shapely >=2.0.6` is required since `eodag 3.0.0`, see https://github.com/CS-SI/eodag/pull/1303.
Please try updating `shapely` and try again

**Reply by sbrunato** [reply]
and if you are using conda, `eodag 3.0.1` is now also available on `conda-forge`

**Reply by gg-bb** [reply]
Ok thank you for your advice. I updated shapely and some other additional packages and it's working now. 
Thank you for your help!
Best Regards
GB
